assets/presto_clp/main.py

```
@@ -0,0 +1,109 @@
+    @property
+    def mount_points(self):
+        return {
+            f"{self.script_dir}/include": "/home/include",
+            CLP_PRESTO_HOST_STORAGE: CLP_PRESTO_CONTAINER_STORAGE,
+        }
+
+    def launch(self):
+        """
+        Runs the benchmarked tool
+        """
+        os.system(f"{CLP_PRESTO_HOST_STORAGE}/sbin/stop-clp.sh -f")
```

Ruff (0.12.2)

Starting a process with a shell, possible injection detected

(S605)